sdk/CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - `OutputData::metadata` changed from `OutputMetadataDto` to `OutputMetadata`;
+- `Client::get_outputs()` and derived methods return `OutputWithMetadata` instead `OutputWithMetadataDto`
```

Comment:
I think it accidentally a word here

Reply:
Do you mean "instead **of**...  and **;** at the end? Or something else? :)

Reply:
I meant the "of" 😆